registration/urls.py

```diff
@@ -1,42 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# OpenCraft -- tools to aid developing and hosting free software projects
-# Copyright (C) 2015-2019 OpenCraft <xavier@opencraft.com>
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-"""
-URL Patterns for the `registration` app
-"""
-
-# Imports #####################################################################
-
-from django.conf.urls import url
-from django.contrib.auth.views import LoginView, LogoutView
-from django.views.generic import TemplateView
-
-from registration.forms import LoginForm
-from registration.views import BetaTestApplicationView
-
-
-# URL Patterns ################################################################
-
-app_name = 'registration'
-urlpatterns = [
-    url(r'^$', BetaTestApplicationView.as_view(), name='register'),
-    url(r'^login/$', LoginView.as_view(authentication_form=LoginForm), name='login'),
```

Comment:
Login and logout views are still required.
You can move those views to `/login` and `/logout`, outside the registration URLs, and then make `/` redirect to the login view.

Can you check if the privacy and ToS pages are not used before erasing them?

Reply:
I checked the TOS and privacy pages in current deployed code, they are not been used.
>https://console.opencraft.com/terms
>https://console.opencraft.com/privacy
I went to the above URLs for checking. So, I removed them.

I took the `login` and `logout` out of the registration URLs.

Reply:
I am resolving this one, as I moved the `login` and `logout` out :+1: